sdk/storage/storage-blob/gulpfile.js

```diff
@@ -7,8 +7,8 @@ const zipFileName = `azurestoragejs.blob-${version}.zip`;
 gulp.task("zip", function(callback) {
   gulp
     .src([
-      "browser/azure-storage.blob.js",
-      "browser/azure-storage.blob.min.js",
+      "browser/azure-storage-blob.min.js",
```

Comment:
Please update section "JavaScript Bundle" in "readme.md". Same with other readme.md files for queue/file. Please check any documents points to old bundle name.

Reply:
Thanks for pointing out.
Done - https://github.com/Azure/azure-sdk-for-js/pull/2297/commits/7788fd0185d53fbfe8fce9bf197ff98f9c8c1383